README.md

```diff
@@ -1,7 +1,12 @@
 # composite-run-steps-action-template
 
 This template can be used to quickly start a new custom composite-run-steps action repository.  Click the `Use this template` button at the top to get started.
-
+- [Inputs](#inputs)
+- [Outputs](#outputs)
+- [Example](#example)
+- [Code of Conduct](#code-of-conduct)
+- [License](#license)
```

Comment:
Just out of curiosity, why was `Incrementing the Version` left off of this list?

Reply:
I see it on the Javascript action template so now I'm thinking it was maybe a miss?

Reply:
Because I generated the ToC before I copied that section in. :(